.vscode/settings.json

```diff
@@ -0,0 +1,10 @@
+{
+  "liveServer.settings.port": 5501,
+  "editor.defaultFormatter": "stylelint.vscode-stylelint",
+  "[html]": {
+    "editor.defaultFormatter": "vscode.html-language-features"
+  },
+  "[javascript]": {
+    "editor.defaultFormatter": "dbaeumer.vscode-eslint"
```

Comment:
- [ ] [OPTIONAL] Please kindly add this file to the `.gitignore` file because it is not directly related to the project you are working on 🙏🏻.

Reply:
Noted. Thank you!